src/components/compounds/CommentsView/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import { getComments, clearComment } from 'Actions/commentActions';
+import { withRouter } from 'react-router';
+
+import './style.scss';
+import PropTypes from 'prop-types';
+import CommentCard from 'Atoms/CommentCard';
+
+export class CommentsView extends React.Component {
+  constructor(props) {
```

Comment:
same as comment above

Reply:
Alright, Thanks.
I will sure see to that. 👍